presto-main/src/test/java/com/facebook/presto/operator/project/TestPageProcessor.java

```diff
@@ -332,25 +332,31 @@ public void testYieldProjection()
         DriverYieldSignal yieldSignal = new DriverYieldSignal();
         PageProcessor pageProcessor = new PageProcessor(
                 Optional.empty(),
-                Collections.nCopies(columns, new YieldPageProjection(new InputPageProjection(0, VARCHAR), yieldSignal)));
+                Collections.nCopies(columns, new YieldPageProjection(new InputPageProjection(0, VARCHAR))));
 
         Slice[] slices = new Slice[rows];
         Arrays.fill(slices, Slices.allocate(rows));
         Page inputPage = new Page(new SliceArrayBlock(slices.length, slices));
 
         PageProcessorOutput output = pageProcessor.process(SESSION, yieldSignal, inputPage);
+
+        // Test yield signal works for page processor
+        // The purpose of this test is NOT to test the yield signal in page projection; we have other test to cover that
+        // In page processor, we check yield signal after a column has been completed processed
+        // So we want to set yield signal when the column has just finished processing in order to let page processor capture the yield signal when the block is returned
+        // Also, we would like to reset the yield signal before starting to process the next column in order NOT to yield per position inside the column
```

Comment:
Add period (.) to the end of each line.

completed -> completely
remove "want to" "would like to"